mig/shared/configuration.py

```diff
@@ -223,6 +223,7 @@ def fix_missing(config_file, verbose=True):
         'trac_id_field': 'email',
         'migserver_http_url': 'http://%%(server_fqdn)s',
         'migserver_https_url': '',
+        'migserver_server_maxsize': -1,
```

Comment:
As we discussed on chat this variable could be given a more clear name like `webserve_max_bytes` or similar. I'll let you decide if you want and then I'll continue merging soon.